lintipy.py

```diff
@@ -183,6 +180,19 @@ def get_env(self):
         ])
         return env
 
+    def set_status(self, state, description, target_url=None):
+        if state not in STATUS_STATES:
+            raise ValueError("%r is not a valid state" % state)
+        data = {
+            'context': self.label,
+            'state': state,
+            'description': description,
+        }
+        if target_url:
+            data['target_url'] = target_url
```

Comment:
DId you try just putting `'target_url': target_url` in the dict initialization? I an imaging, if GitHub API would receive `target_url=None`, it would not blow up, but we can get rid of extra condition here. :)